README.md

```diff
@@ -1,268 +1,78 @@
-
-Before you begin writing, make sure to read the [article style guide](#article-style-guide) below.
+This repo is where we plan and maintain these Guide articles, which we then host on our community's wiki-like Guide website at [`guide.freecodecamp.org`](https://guide.freecodecamp.org).
 
-## How to contribute
-You can create a PR (Pull Request) with your draft article (or edits on an existing article) in two ways:
-
-1) The easiest method is to use the GitHub interface. Watch the video demonstration or follow the steps below it:
+## What are Guide articles?
 
-![Gif showing the GitHub interface steps](https://i.imgur.com/0cmxJwN.gif)
+Guide articles can be an explanation of a syntax, design pattern, what aria labels are for, or something like what the numbers mean in the top right-hand corner of your screen when at [freecodecamp.org](https://freecodecamp.org).
 
-- Go into the "pages" folder (located in `guides/src`) and find the article stub you'd like to write or edit. All stubs will be in an index.md file
-- Click the "Edit this file" pencil icon and make your changes to the file in GitHub-flavored Markdown
-- Scroll to the bottom of the screen and add a commit message explaining your changes. Then select the radio button option for "Create a new branch for this commit and start a pull request" and click "Propose file changes"
-- On the next screen, you can add any other details about your PR, then click "Create pull request"
+You can find an [example article about HTML Elements here](./src/pages/html/elements/index.md).
 
+## Installation
 
-2) If you prefer to write locally before submitting a PR, then follow these steps:
-- Fork this repository
-- Copy it to your local machine by running the command `git clone https://github.com/YOUR-GITHUB-USERNAME/guides.git`
-- Add a remote upstream so git knows where the official freeCodeCamp guides repository is located by running the command `git remote add upstream https://github.com/freeCodeCamp/guides.git`
-- Create a new branch for your work with the command `git checkout -b NEW-BRANCH-NAME`. Try to name your branch in a way that describes your article topic, like `fix/ArticleHTMLElements`
-- Write your article, commit your changes locally with the command `git commit -m "SHORT MESSAGE"`, and push your new branch to GitHub with the command `git push origin NEW-BRANCH-NAME`
-- Go to your repository on GitHub and open a PR
+If you want to run a version of the guides repository locally, follow these steps:
 
-Make sure to maintain your local fork going forward so it stays up-to-date with the freeCodeCamp guides repository. The next time you want to contribute, checkout your local `master` branch and run the command `git pull --rebase upstream master` before creating a new branch. This will grab all the changes on the official `master` branch without making an additional commit in your local repository.
+### Fork this repository.
```

Comment:
This section is redundant - it's covered in Contributing.md

Reply:
I have removed that section.